_shared/hooks/post_gen_project.py

```diff
@@ -45,6 +45,9 @@ def remove_conditional_files():
 
     {% if cookiecutter.get("linter") == "ruff" %}
     paths_to_remove.extend(["tests/pyproject.toml"])
+    {% if cookiecutter._directory in ['pyapp', 'pyramid-app'] %}
+    paths_to_remove.extend(["setup.cfg"])
```

Comment:
Now that we're no longer using pycodestyle the `setup.cfg` file is no longer needed. We do still need `setup.cfg` in packages because it [contains a bunch of package metadata](https://github.com/hypothesis/test-pypackage/blob/c65d559fbe4cf915830adfd4a314a6494e0564a1/setup.cfg#L1-L29), though I wonder whether that could be moved into `pyproject.toml` nowadays (separate PR).